### ISSUE TITLE
null converts to Any

### ISSUE THREAD

**Issue by jessemyers-lettuce** [MarcoMuellner/openapi-python-generator#52]
OpenAPI type `null` is converted to Python type `Any`, most likely [here](https://github.com/MarcoMuellner/openapi-python-generator/blob/8d1d695917221de402f17f7be7bacfce9cf2cd03/src/openapi_python_generator/language_converters/python/model_generator.py#L166):

```py
    elif schema.type is None or schema.type == "null":
        converted_type = pre_type + "Any" + post_type
```

In Fast API (probably among other sources), `Foo | None` is expressed as:

```json
"anyOf": [
      {
        "$ref": "#/components/schemas/Foo"
      },
      {
        "type": "null"
      }
    ]
```

because of this issue, we end up with a generated type that uses `Union[Foo | Any]` instead of `Union[Foo | None]`

If this is intentional, I'd love to understand why so I can create a fix that doesn't break some implicit, non-obvious behavior.

**Comment by jessemyers-lettuce** [comment]
For my usage, the fix in #53 works.

**Comment by MarcoMuellner** [comment]
Fixed in #53
